src/sentry/api/serializers/models/project_key.py

```diff
@@ -100,4 +103,8 @@ def serialize(
                 "hasDebug": get_dynamic_sdk_loader_option(obj, DynamicSdkLoaderOption.HAS_DEBUG),
             },
         }
+
+        if user.is_superuser:
```

Comment:
Should this be `is_active_superuser` as well?

Reply:
Thanks, passed in the request to the serializer now to call `is_active_superuser`.